.github/workflows/release.yml

```diff
@@ -16,3 +18,13 @@ jobs:
         env:
           STDLIB_TOKEN: ${{ secrets.STDLIB_TOKEN }}
 
+  homebrew:
+    name: Bump Homebrew formula
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@eb843fa04b91b8bc5d4953899e3fff15c95f347c
+        if: "!contains(github.ref, '-')" # skip prereleases
```

Comment:
Should this also be added to the documentation build job?

Reply:
I don't really understand it. I was copy and paste magic there. We don't have prereleases at this point (that I am aware of). So I don't think it is needed. Should I delete it?

Reply:
It seems that this condition would only invalidate tags with a `-` (for example, `0.34.0-rc1`). I would remove it to keep it consistent with the other job, since we don't really need it.

Reply:
Updated